src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java

```diff
@@ -150,6 +150,11 @@ public static MultipleClassification of(String strategy) {
     private Integer numKFold = -1;
 
     /**
+     * Random seed is used to split training set and test set, default value is 1500890350L.
+     */
+    private Long baggingSampleSeed = Long.valueOf(1500890350L);
```

Comment:
I would suggest set it to -1 as default, if it is -1, in each worker, set logic as before by using new Random(). Better?